content/blog/zardshard/2023-05-05_[gsoc_2023]_plans_for_improving_icon-o-matic.md

```diff
@@ -0,0 +1,47 @@
++++
+type = "blog"
+title = "[GSoC 2023] Plans for improving Icon-O-Matic"
+author = "Zardshard"
+date = "2023-05-05 14:07:09-04:00"
+tags = ["haiku", "software", "icon-o-matic", "gsoc", "gsoc2023"]
```

Comment:
Is the icon-o-matic tag here ok, or should it be spelled differently (e.g. Icon-O-Matic)?

Reply:
looks like all lowercase is OK for tags